todoman/cli.py

```diff
@@ -290,6 +302,7 @@ def move(ctx, list, ids):
 @click.option('--location', help='Only show tasks with location containg TEXT')
 @click.option('--category', help='Only show tasks with category containg TEXT')
 @click.option('--grep', help='Only show tasks with message containg TEXT')
+# XXX: shouldn't this be an ngarg-1 this?:
```

Comment:
As a follow-up: I removed this. `option` can't have `nargs=-1`, only `argument` does.